backend/app/routers/files.py

```diff
@@ -170,6 +179,16 @@ async def update_file(
             creator=user,
         )
         await db["file_versions"].insert_one(new_version.to_mongo())
+        # Update entry to the file index
+        doc = {
+            "doc": {
+                "name": updated_file.name,
+                "creator": updated_file.creator.email,
+                "created": datetime.utcnow(),
+                "download": updated_file.downloads,
```

Comment:
I added a few more required fields for the files in another PR. This might be updated...

Reply:
we can merge this first and then i'll add more fields in my other PR. Looks like this is a smaller PR so it might be easier to be merged in first